mllib/src/main/scala/org/apache/spark/mllib/evaluation/MulticlassMetrics.scala

```diff
@@ -151,6 +151,14 @@ class MulticlassMetrics @Since("1.1.0") (predictionAndLabels: RDD[(Double, Doubl
   lazy val recall: Double = precision
 
   /**
+   * Returns accuracy
+   * (equals to precision for multicalss classifier
+   * because it is tp/labelCount)
+   */
+  @Since("2.0.0")
+  lazy val accuracy: Double = precision
```

Comment:
@srowen Thanks for your review! I think it makes a lot sense of deprecating "precision" with "accuracy" as I got confused when I tried to add accuracy yesterday. I will make the changes.